ServerCore/Pages/Events/Import.cshtml.cs

```diff
@@ -56,6 +57,20 @@ public async Task<IActionResult> OnPostImportAsync()
 
             using (var transaction = _context.Database.BeginTransaction(System.Data.IsolationLevel.Serializable))
             {
+                // Step 0: Make sure all admins exist
+                foreach (var sourceEventAdmin in sourceEventAdmins)
+                {
+                    var destEventAdmin = await _context.EventAdmins.Where((e) => e.Event == Event && e.Admin == sourceEventAdmin.Admin).FirstOrDefaultAsync();
```

Comment:
It might be better to get the list of destination event admins first to avoid the n^2 queries